extensions/resource-deployment/src/ui/resourceTypePickerDialog.ts

```diff
@@ -36,6 +37,12 @@ export class ResourceTypePickerDialog extends DialogBase {
 		this._selectedResourceType = resourceType;
 		this._dialogObject.okButton.label = localize('deploymentDialog.OKButtonText', 'Select');
 		this._dialogObject.okButton.onClick(() => this.onComplete());
+		this._installToolButton = azdata.window.createButton(localize('deploymentDialog.InstallToolsButton', 'Install tools'));
```

Comment:
perhaps call it 'install dependencies'

Reply:
I'll leave it as is since we use 'tools' rest of places.